code/game/objects/structures/pinatas.dm

```diff
@@ -0,0 +1,83 @@
+/obj/structure/pinata
+	name = "corgi pinata"
+	desc = "A paper mache representation of a corgi that contains all sorts of sugary treats."
```

Comment:
```suggestion
	desc = "A papier-mâché representation of a corgi that contains all sorts of sugary treats."
```

otherwise you pronounce it "paper mash" and that sounds wrong. DM should support this (i know we use the é for québécois)